Find-Package.NuGet.ArgumentCompleters.ps1

```diff
@@ -0,0 +1,84 @@
+<#PSScriptInfo
```

Comment:
Consider having no dots in the file name. It is recommended to follows the cmdlet naming guidelines for the script file name as user invokes the script files as command after installing it.

Reply:
Renaming to Register-FindPackageNuGetArgumentCompleter.ps1

---
In reply to: [154492090](https://github.com/OneGet/NuGetProvider/pull/25#discussion_r154492090) [](ancestors = 154492090)